metashare/api/models.py

```diff
@@ -447,6 +447,16 @@ def finalize_commit_changes(self, error=None):
                 self, error, "SCRATCH_ORG_COMMIT_CHANGES_FAILED"
             )
 
+    def remove_scratch_org(self):
+        from .serializers import ScratchOrgSerializer
+
+        payload = ScratchOrgSerializer(self).data
+        message = {"type": "SCRATCH_ORG_REMOVED", "payload": payload}
+        async_to_sync(push.push_message_about_instance)(self, message)
+        # Use super().delete to avoid accidentally sending a
+        # SCRATCH_ORG_DELETE event:
+        super().delete()
```

Comment:
Maybe it would be clearer to have named methods for these? Or a flag we can pass to one method that signifies whether to notify?

Reply:
We overrode `.delete` because DRF uses it in the views; I think this is the lesser evil.

Reply:
The flag idea could be good though.